assets/js/modules/adsense/components/setup/v2/SetupAccountCreateSite.js

```diff
@@ -21,8 +21,67 @@
  */
 import PropTypes from 'prop-types';
 
+/**
+ * WordPress dependencies
+ */
+import { Fragment, useCallback } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import Data from 'googlesitekit-data';
+import Button from '../../../../../components/Button';
+import { MODULES_ADSENSE } from '../../../datastore/constants';
+import { ErrorNotices } from '../../common';
+import { trackEvent } from '../../../../../util';
+import useViewContext from '../../../../../hooks/useViewContext';
+const { useSelect } = Data;
 export default function SetupAccountCreateSite( { accountID } ) {
-	return <div>TODO: UI to create a new site in account { accountID }</div>;
+	const viewContext = useViewContext();
+
+	const addSiteURL = useSelect( ( select ) =>
+		select( MODULES_ADSENSE ).getServiceAccountManageSiteURL()
+	);
+
+	const addSiteHandler = useCallback(
+		( event ) => {
+			event.preventDefault();
+			trackEvent( `${ viewContext }_adsense`, 'create_site' );
+			global.open( addSiteURL, '_blank' );
+		},
+		[ addSiteURL, viewContext ]
+	);
+
+	if ( ! accountID || ! addSiteURL ) {
+		return null;
+	}
```

Comment:
I think this is appropriate logic-wise, but do we want to have explicit checks for `undefined` here for `addSiteURL` instead of the false-y check? Just to clarify the intent, because otherwise addSiteURL shouldn't even be false-y once the `select( MODULES_ADSENSE ).getAccountID()` and `select( CORE_SITE ).getReferenceSiteURL()` resolve, right? 🤷🏻‍♂️

Reply:
This condition was simply copied from the V1 [`SetupSiteAdd`](https://github.com/google/site-kit-wp/blob/develop/assets/js/modules/adsense/components/setup/SetupSiteAdd.js#L50) component, as per the IB, however I do agree with your suggestion to test for `undefined` here to better indicate the intent. I've made that change, but won't modify `SetupSiteAdd`, that component will be removed fairly soon anyway when we strip out the `adsenseServiceV2` feature flag...